cli/tests/environment-pull.bats

```diff
@@ -84,17 +84,6 @@ function update_dummy_env() {
 
 # ---------------------------------------------------------------------------- #
 
-
-# bats test_tags=pull:l1
-@test "l1: pull login: running flox pull before user has login metadata prompts the user to login" {
-
-  unset FLOX_FLOXHUB_TOKEN; # logout, effectively
-
-  run "$FLOX_BIN" pull --remote owner/name # dummy remote as we are not actually pulling anything
-  assert_failure
-  assert_output --partial 'Please login to floxhub with `flox auth login`'
-}
-
```

Comment:
question blocking: is there coverage for both an authenticated and unauthenticated pull?

Reply:
pull works regardless of being logged in.
The tests are running with a token set

Reply:
So shouldn't we run a test without a token set?

Reply:
it doesnt make a difference in these tests anyway as the token is not not used for authentication (we're pulling from a bare repo).
But i can add a test showing that we dont block on it..